python/paddle/fluid/dygraph/dygraph_to_static/loop_transformer.py

```diff
@@ -309,11 +316,59 @@ def _node_var_type_is_basic(self, node_var_type):
         return False
 
     def _is_call_func_name_node(self, node):
-        parent_node = self.node_to_wrapper_map[node].parent.node
+        parent_node = self._get_parent_node(node)
         if isinstance(parent_node, gast.Call) and parent_node.func == node:
             return True
         return False
 
+    def _get_parent_node(self, node):
+        wrapper_node = self.node_to_wrapper_map.get(node)
+        if wrapper_node:
+            parent_node = wrapper_node.parent.node
+            return parent_node
+        return None
+
+    def _remove_target_vars_of_for(self,
+                                   before_or_after_loop_vars,
+                                   loop_node=None):
+        """
```

Comment:
`loop_node` has default value `None`, but no usage with `None` appears in this PR. What is the behavior if we set it `None`?

Reply:
The default value of `loop_node` is not necessary, I will remove it